setup.py

```diff
@@ -176,6 +175,10 @@ def run(self):
         execute(self, post_install, [self])
 
 
+control_node_serial_data = 'control_node_serial/control_node_serial_interface'
```

Comment:
The variable name is not very related to its content (more related to serial interface). By the way, you don't really need an extra variable here, just split the line below